webpack/assets/javascripts/react_app/components/common/EmptyLine/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Skeleton from 'react-loading-skeleton';
+import { translate as __ } from '../../../../react_app/common/I18n';
+
+const EmptyLine = ({ isLoading }) => {
+  if (isLoading) {
+    return <Skeleton />;
+  }
+  return __('N/A');
```

Comment:
How about a consumer option? so the `N/A` would be the default value, but the caller can override it with another value/component (i.e Empty state component)